pkg/install/installer_test.go

```diff
@@ -182,15 +182,15 @@ var _ = Describe("Profile", func() {
 		Expect(os.RemoveAll(rootDir)).To(Succeed())
 	})
 
-	Context("Make", func() {
+	Context("Install", func() {
 		It("generates the artifacts", func() {
-			maker := profile.NewProfilesArtifactsMaker(profile.MakerConfig{
+			installer := install.NewInstaller(install.InstallerConfig{
```

Comment:
this feels frigging weird. waaay to many `install`s. :D Does config need to be `InstallerConfig`? Do we have another config in that package? I don't think we do.

Reply:
yeah I think jsut calling it `Config` makes sense, then tis `install.config`